application/pipeline-stacks/oncoanalyser/assets/run.sh

```diff
@@ -495,17 +495,17 @@ fi
 # NOTE(SW): using new conditional block to separate functionality
 nextflow_args=''
 if [[ ${mode} == 'wgs' ]]; then
-  nextflow_args='--processes_exclude isofox,orange'
+  nextflow_args='--processes_exclude isofox'
 elif [[ ${mode} == 'wts' ]]; then
   nextflow_args='--mode manual --processes_include isofox,cuppa'
 elif [[ ${mode} == 'wgts' ]]; then
-  nextflow_args='--processes_exclude orange'
+  :
```

Comment:
You can also do the following to make it a little clearer that ':' means the equivalent of 'pass'

```
: '
pass
'
```

Reply:
Nice improvement, thanks